AttitudeManager/FlightModes/Src/fbwa.cpp

```diff
@@ -0,0 +1,38 @@
+#include "fbwa.hpp"
+#include "CommonDataTypes.hpp"
+#include "AM.hpp"
+#include "config.hpp"
+#include <cassert>
+
+namespace AM {
+    AttitudeManagerInput FBWA::run(const AttitudeManagerInput& input) {
+        
+        AttitudeManagerInput mappedOutputs;
+
+        assert(AM::AttitudeManager::INPUT_MAX == 100 && AM::AttitudeManager::INPUT_MIN == -100);
```

Comment:
```suggestion
        assert(input.pitch <= INP_MAX && input.pitch >= INP_MIN);
```

Or something similar.

You just want to make sure that your inputs are within -100 to +100, and that it's an integer.

For this PR (just so that camron has something to work off of) - we can ignore it.